### PR TITLE
Add DeleteEntryCommandParserTest

### LINKED ISSUE #172: Tests for delete entry command

### PR DESCRIPTION
**Pull request by lrj689**

### CODE REVIEW

**Review by goatygoatygoat — APPROVED** [review]

File: src/test/java/seedu/address/logic/parser/DeleteEntryCommandParserTest.java [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    @Test
+    public void parse_validArgs_returnsDeleteEntryCommand() {
+        assertParseSuccess(parser, "1", new DeleteEntryCommand(INDEX_FIRST));
+    }
+
+    @Test
+    public void parser_invalidArgs_throwsParseException() {
+        assertParseFailure(parser, "name",
+                String.format(MESSAGE_INVALID_COMMAND_FORMAT, DeleteEntryCommand.MESSAGE_USAGE));
+    }
```
Comment (tsh22) [inline src/test/java/seedu/address/logic/parser/DeleteEntryCommandParserTest.java:25]: Maybe can add in testing for out of range indices

Comment (tsh22) [inline src/test/java/seedu/address/logic/parser/DeleteEntryCommandParserTest.java:25]: Maybe can add in testing for out of range indices

**Review by tsh22 — CHANGES_REQUESTED** [review]

**Review by tsh22 — APPROVED** [review]

**Review by KevinLohJunYong — APPROVED** [review]
